composer.lock

```diff
@@ -418,16 +538,16 @@
         },
         {
             "name": "simplesamlphp/simplesamlphp",
-            "version": "v1.17.7",
+            "version": "v1.18.0",
```

Comment:
Looks like we pulled in 1.18.0. We probably want to update our [version specifier](https://getcomposer.org/doc/articles/versions.md#tilde-version-range-) to `~1.17.8` since simplesamlphp sometimes includes breaking changes in minor version updates.

FWIW, we will want to move to 1.18 at some point, but not necessary as part of this security fix.

Reply:
OK, resolved and force-pushed... Give it 👀 again?